docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -15,114 +15,1015 @@
     specific language governing permissions and limitations
     under the License.
 
+###########################
+KubernetesPodOperator Guide
+###########################
+
+.. contents:: :local:
 
 
 .. _howto/operator:KubernetesPodOperator:
 
-KubernetesPodOperator
-=====================
+Kubernetes Pod Operator "Hello World!"
+======================================
+
+- Creating Task using KubernetesPodOperator which prints "Hello World [HOSTNAME]".  Add ``in_cluster=False`` to
+  KubernetesPodOperator constructor when running examples with minikube.
+
+
+.. code-block:: python
+
+  with DAG(dag_id="example_k8s_operator", start_date=days_ago(1),
+           schedule_interval='@once', tags=["example"]) as dag:
+      task1 = KubernetesPodOperator(task_id='k8s_pod_operator_task',
+                                    name='airflow_pod_operator',
+                                    namespace='default',
+                                    image='alpine',
+                                    cmds=["sh", "-c",
+                                          'echo "Hello World from pod [$HOSTNAME]"'],
+                                    startup_timeout_seconds=60,
+                                    )
+
+- Executing / Debugging example and checking the logs
```

Comment:
Is there something you were going to write here? I just see logs...

Reply:
I meant after executing the dag with configuration, i will update the line.